[2025-02-11T00:12:22.281+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T00:12:22.314+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-11T00:08:07.764104+00:00 [queued]>
[2025-02-11T00:12:22.329+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-11T00:08:07.764104+00:00 [queued]>
[2025-02-11T00:12:22.330+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-02-11T00:12:22.358+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-02-11 00:08:07.764104+00:00
[2025-02-11T00:12:22.368+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4118) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T00:12:22.371+0000] {standard_task_runner.py:72} INFO - Started process 4119 to run task
[2025-02-11T00:12:22.373+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-02-11T00:08:07.764104+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpspz5fvgr']
[2025-02-11T00:12:22.376+0000] {standard_task_runner.py:105} INFO - Job 169: Subtask assign_permissions_dag
[2025-02-11T00:12:22.496+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-11T00:08:07.764104+00:00 [running]> on host 9c5f21acce5a
[2025-02-11T00:12:22.707+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T00:08:07.764104+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-11T00:08:07.764104+00:00'
[2025-02-11T00:12:22.709+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-11T00:12:22.714+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-11T00:12:22.738+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-11T00:12:22.739+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-02-11T00:08:07.764104+00:00, execution_date=20250211T000807, start_date=20250211T001222, end_date=20250211T001222
[2025-02-11T00:12:22.791+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T00:12:22.825+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-11T00:12:22.828+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
